configure.ac

```diff
@@ -1106,6 +1106,23 @@ OPAL_MCA
 
 m4_ifdef([project_ompi], [OMPI_REQUIRE_ENDPOINT_TAG_FINI])
 
+# Last minute disable of Open SHMEM if we didn't find any oshmem SPMLs
+if test "$project_oshmem_amc" = "true" -a $OSHMEM_FOUND_WORKING_SPML -eq 0 ; then
```

Comment:
Should this (and the other lines like this in this commit) be `if test ... && test ...`?  I think there's some BSD `test` that doesn't like `-a`/`-o` -- I remember we've gotten in portability kerfuffles with this before.